src/builder/app.js

```diff
@@ -1,13 +1,14 @@
-/*jslint node:true, strict:false*/
+/*jshint node:true, strict:false*/
```

Comment:
Want to remove this all together from source files. jshintrc would be better I think - what do you think?